tun_tap/kill_modem.sh

```diff
@@ -2,3 +2,6 @@
 
 PID=$(pidof modemd)
 kill -9 $PID 
+
+PID=$(pidof wpa_supplicant)
+kill -9 $PID 
```

Comment:
Did you mean to commit this? This doesn't have anything to do with the IP fragmentation issue described in the commit log.

Reply:
Yes I meant to add that as well. wpa_supplicant should stop running when you stop the modem since it requires the adi_radio device